ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -27,6 +27,11 @@
   INTERNAL_ERROR(-32603, "Internal error"),
   METHOD_NOT_ENABLED(-32604, "Method not enabled"),
 
+  // eth_sendTransaction specific error message
+  ETH_SEND_TX_NOT_AVAILABLE(
+      -32604,
+      "The method eth_sendTransaction is not supported. Use eth_sendRawTransaction with a supported signer, e.g. ethsigner."),
```

Comment:
I don't think this is quite right.  Your options are: 
* Use eth_sendRawTransaction with Pantheon 
* Use eth_sendTransaction with a supported signer (eg, EthSigner) 

You can send eth_sendRawTransaction through EthSigner but there's no benefit because the transaction would already be signed.